src/helpers/utils.tsx

```diff
@@ -986,3 +988,18 @@ export type DefaultDrillDownPropsType = Pick<
 export const getPlanStatusToDisplay = (planStatuses: string[]): string[] => {
   return Object.values(PlanStatus).filter(status => !planStatuses.includes(status));
 };
+
+/**
+ *
+ * @param {string | number} value - string date
+ * @param {string} dateFarmat - format of returned date
+ * @param {string} fallbackText fall back value when string date not valid
+ */
+export const formatDates = (
+  value: string | number,
+  dateFormat: string,
+  fallbackText: string = INVALID_DATE
```

Comment:
Does this mean that the string displayed by default will be "Invalid date"?

Reply:
Yes

Reply:
Is there a place in the code where we override it to show a dash like we had talked about?

Reply:
They have custom strings i.e `Not visited` and `No decision form` by the time we discussed i had not realised this.

Reply:
@ciremusyoka will this PR then address the issue of "invalid date" appearing on the reports?

Reply:
@moshthepitt yes it addresses that. Now we have `Not visited` or `No decision form` instead of `invalid date`. Check on the screenshot posted above.